test/lit/wasm-split/invalid-options.wast

```diff
@@ -33,7 +33,7 @@
 ;; RUN: not wasm-split %s --instrument --split-funcs=foo 2>&1 \
 ;; RUN:   | filecheck %s --check-prefix INSTRUMENT-SPLIT-FUNCS
 
-;; --instrument is required to use -o
+;; --instrument requires -o1 and -o2 rather than -o
```

Comment:
Sorry, but isn't this the opposite? `--instrument` requires `-o` and not having `--instrument` requires `-o1` and `-o2`..

Reply:
Gentle ping :) This is a pretty minor question, but just in case you haven't had a chance to see it. (The same for the question above)

Reply:
Sorry for missing these questions, and thank you for your fix!